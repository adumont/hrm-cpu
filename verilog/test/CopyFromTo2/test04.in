18 d1
48 5c
17 3f
19 3b
22 ee
50 39
10 fe
51 b0
24 92
47 d1
35 5b
32 af
06 bf
51 b5
50 31
49 d2
20 8b
09 e8
14 74
56 07
31 db
31 9d
28 8b
02 07
49 bf
56 6e
03 9c
44 84
45 97
40 53
09 9a
28 22
39 8f
18 9f
05 f2
10 73
06 bd
47 bd
13 a9
26 71
10 ae
18 67
34 fe
26 35
54 47
11 d5
44 22
31 9f
