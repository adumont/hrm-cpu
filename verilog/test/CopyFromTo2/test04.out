d1
5c
3f
3b
ee
39
fe
b0
92
d1
5b
af
bf
b5
31
d2
8b
e8
74
07
db
9d
8b
07
bf
6e
9c
84
97
53
9a
22
8f
9f
f2
73
bd
bd
a9
71
